app/src/common/message_channel.h

```diff
@@ -88,7 +88,9 @@ ZBUS_CHAN_DECLARE(
 	NETWORK_CHAN,
 	PAYLOAD_CHAN,
 	TIME_CHAN,
-	TRIGGER_CHAN
+	TRIGGER_CHAN,
```

Comment:
Remove

Reply:
Other modules still use it. I just modified a couple of modules to show an alternative way of triggering without sending a message.